examples/rhpam/springboot.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.redhat.io/rhpam-7/rhpam-kogito-runtime-jvm-rhel8:7.12.0
+FROM ibm-bamoe/bamoe-kogito-runtime-jvm-rhel8:7.12.0
```

Comment:
Shouldn't it be `registry.connect.redhat.com/ibm-bamoe/bamoe-kogito-runtime-jvm-rhel8:8.0.0`?

Reply:
+1